src/components/SortBy.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react'
+import { sorteArticlesBy } from '../utils/api';
+
+function SortBy({setArticles}) {
+    const [ sortValue, setSortValue] = useState('');
+    
+    useEffect(() => {
+        
+    }, [])
+    
+    const handleSort = (e) => {
+        let value = e.target.value;
+        console.log(value, ' << value')
+
+        sorteArticlesBy(value).then((response) => {
+            setSortValue(response.data);
+            setArticles(response);
+        })
+        .catch((err) => {
+            console.log(err, ' << the err');
+        })
+    }
+    
+    return (
+        <div>
+            <select className='sortby-box' value={sortValue} onChange={handleSort}> 
+                <option> select sort preference </option>   
```

Comment:
You don't need an option that has no value; trust your users to know how to select an option from a dropdown :D